src/napari_matplotlib/tests/scatter/test_scatter.py

```diff
@@ -23,3 +24,27 @@ def test_scatter_2D(make_napari_viewer, astronaut_data):
     viewer.layers.selection.add(viewer.layers[0])
     viewer.layers.selection.add(viewer.layers[1])
     return deepcopy(fig)
+
+
+@pytest.mark.mpl_image_compare
+def test_scatter_3D(make_napari_viewer, brain_data):
+    viewer = make_napari_viewer()
+    widget = ScatterWidget(viewer)
+    fig = widget.figure
+
+    viewer.add_image(brain_data[0], **brain_data[1], name="brain")
+
+    viewer.add_image(
+        brain_data[0] * -1, **brain_data[1], name="brain_reversed"
+    )
+    # De-select existing selection
+    viewer.layers.selection.clear()
+    # pdb.set_trace()
+    axis = viewer.dims.last_used
+    slice_no = array(brain_data[0]).shape[0] - 1
```

Comment:
```suggestion
    slice_no = brain_data[0].shape[0] - 1
```
Isn't `brain_data` already an array?